lib/archethic_web/router.ex

```diff
@@ -71,6 +71,8 @@ defmodule ArchEthicWeb.Router do
       :last_transaction_content
     )
 
+    get("/origin_key/:origin_public_key", ArchEthicWeb.API.TransactionController, :origin_key)
```

Comment:
Why do you want to use the TransactionController ?

Reply:
Should I create a new API Controller ?

Reply:
An OriginKeyController could be great

Reply:
Done !